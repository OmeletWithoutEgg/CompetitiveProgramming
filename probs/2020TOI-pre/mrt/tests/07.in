16
23 21 11 19 1 16 2
13 9 1 10 5 2 12
23 15 5 11 12 6 14
59 44 1 32 4 6 56
13 8 10 0 7 3 10
7 4 6 5 5 2 6
2 0 1 0 1 1 0
59 31 43 3 35 43 42
73 66 41 31 59 29 8
29 4 23 12 28 26 19
73 57 61 45 26 41 12
19 13 7 10 15 15 9
41 35 24 10 12 35 34
37 24 31 14 22 24 36
7 0 2 6 1 5 1
2 1 1 0 1 1 1
