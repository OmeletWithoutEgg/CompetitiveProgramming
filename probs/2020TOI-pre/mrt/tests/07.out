6 13
4 7
8 4
25 18
11 5
4 3
0 1
20 7
61 21
25 1
54 21
0 8
28 18
35 7
3 1
1 1
